cheroot/ssl/__init__.py

```diff
@@ -56,13 +56,27 @@ def _ensure_peer_speaks_https(raw_socket, /) -> None:
 
 
 class Adapter(ABC):
-    """Base class for SSL driver library adapters.
+    """
+    Base class for SSL driver library adapters.
```

Comment:
I wouldn't update this. The original same-line style is a pretty popular convention and its fully PEP 257-compliant.
```suggestion
    """Base class for SSL driver library adapters.
```

Keeping it as is will also make the diffs smaller and thus easier to reason about.